crates/volta-core/src/session.rs

```diff
@@ -108,6 +102,13 @@ impl Session {
         self.project.get()
     }
 
+    /// Returns the user's currently active platform, if any
+    ///
+    /// Active platform is determined by first looking at the Project Platform
+    /// - If it exists and has a Yarn version, then we use the project platform
```

Comment:
Nit: there should be an empty line between the previous line and this one, so that it renders correctly as a Markdown list in editors and IDEs which understand that.

Without, it'll tend to render like this:

> Active platform is determined by first looking at the Project Platform - If it exists and has a Yarn version, then we use the project platform - If it exists but doesn't have a Yarn version, then we merge the two, pulling Yarn from the user default platform, if available - If there is no Project platform, then we use the user Default Platform

With, it'll render as expected:

> Active platform is determined by first looking at the Project Platform
> 
> - If it exists and has a Yarn version, then we use the project platform
> - If it exists but doesn't have a Yarn version, then we merge the two, pulling Yarn from the user default platform, if available
> - If there is no Project platform, then we use the user Default Platform

(There's some variation on how various Markdown engines parse it, so you *may* see it work as expected as is in *some* editors as is, but you *will* see it work as expected in *all* editors with the tweak.)

Reply:
Done!